src/app/services/search/search.service.ts

```diff
@@ -47,4 +47,10 @@ export class SearchService {
     isValidBlock(block: string): boolean {
         return block && block.length === 64;
     }
+
+    isValidBNSDomain(bns: string): boolean {
+        const parts = bns.split('.');
+        //later, can also check for illegal characters once that is more settled
+        return parts.length === 2 && parts[0].length <= 32;
+    }
```

Comment:
Could this be placed in the UtilService & then used here & in the AccountComponent?